### PR TITLE
numa:explicit CPU-NUMA association workaround on arm

### PR DESCRIPTION
**Pull request by zhenyzha**
Explicit CPU-NUMA association workaround on arm

ID: 2060258
Signed-off-by: zhenyzha <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
```
 (01/35) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.numa_memdev_o
ptions.numa_policy.policy_default.prealloc.arm64-pci: PASS (135.34 s)
......
 (35/35) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.migration_with_numa.arm64-pci: PASS (157.69 s)
RESULTS    : PASS 35 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
```

@PaulYuuu Could you pls help review when you are free? thanks.

File: qemu/tests/cfg/balloon_memhp.cfg [hunk 1]
```diff
@@ -23,8 +23,14 @@
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
     threshold = 0.12
-    ppc64,ppc64le:
+    aarch64,ppc64,ppc64le:
         threshold = 0.15
+    # Due to known issue
+    aarch64:
+        only RHEL.9.0
```
Comment (PaulYuuu) [inline qemu/tests/cfg/balloon_memhp.cfg:30]: `9.0.0`

Comment (zhenyzha) [reply qemu/tests/cfg/balloon_memhp.cfg:30]: We don't have `9.0.0` planned so I recommend keeping it.

Comment (zhenyzha) [reply qemu/tests/cfg/balloon_memhp.cfg]: corrected

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**